<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventFilterProviderWidget</class>
 <widget class="QWidget" name="EventFilterProviderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="gbStages">
      <property name="title">
       <string>Stages</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>231</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="tbStageRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QTableView" name="tvStages">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="tbStageAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tbStageEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="gbBlocks">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Blocks</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="3">
        <widget class="QTableView" name="tvBlocks">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="tbBlockAdd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tbBlockRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>172</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="gbEventFilters">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Events filters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" rowspan="4">
        <widget class="QTableView" name="tvEventFilters">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>139</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="tbEventFilterRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="tbEventFilterAdd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tbEventFilterEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tvStages</tabstop>
  <tabstop>tbStageAdd</tabstop>
  <tabstop>tbStageEdit</tabstop>
  <tabstop>tbStageRemove</tabstop>
  <tabstop>tvBlocks</tabstop>
  <tabstop>tbBlockAdd</tabstop>
  <tabstop>tbBlockRemove</tabstop>
  <tabstop>tvEventFilters</tabstop>
  <tabstop>tbEventFilterAdd</tabstop>
  <tabstop>tbEventFilterEdit</tabstop>
  <tabstop>tbEventFilterRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
